File: selector/src/main/java/com/luck/picture/lib/adapter/holder/CameraViewHolder.java
package com.luck.picture.lib.adapter.holder;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.luck.picture.lib.R;
import com.luck.picture.lib.config.SelectMimeType;
import com.luck.picture.lib.config.SelectorProviders;
import com.luck.picture.lib.style.SelectMainStyle;
import com.luck.picture.lib.utils.StyleUtils;

/**
 * @author：luck
 * @date：2021/11/20 3:54 下午
 * @describe：CameraViewHolder
 */
public class CameraViewHolder extends BaseRecyclerMediaHolder {

    public CameraViewHolder(View itemView) {
        super(itemView);
        ImageView tvCamera = itemView.findViewById(R.id.tvCamera);
        selectorConfig = SelectorProviders.getInstance().getSelectorConfig();
        SelectMainStyle adapterStyle = selectorConfig.selectorStyle.getSelectMainStyle();
        int background = adapterStyle.getAdapterCameraBackgroundColor();
        if (StyleUtils.checkStyleValidity(background)) {
            tvCamera.setBackgroundColor(background);
        }
    }

}

File: tap-selector/src/main/java/com/luck/picture/lib/adapter/CameraViewHolder.kt
package com.luck.picture.lib.adapter

import android.view.View
import android.widget.ImageView
import com.luck.picture.lib.R
import com.luck.picture.lib.adapter.base.BaseListViewHolder

/**
 * @author：luck
 * @date：2022/11/30 3:29 下午
 * @describe：CameraViewHolder
 */
open class CameraViewHolder(itemView: View) : BaseListViewHolder(itemView) {
    private var tvCamera: ImageView = itemView.findViewById(R.id.tv_camera)
    fun bindData(position: Int) {
        itemView.setOnClickListener {
            if (mItemClickListener != null) {
                mItemClickListener?.openCamera()
            }
        }
    }
}